### PR TITLE
✅ Add output path to scaffold application

### PR DESCRIPTION
**Pull request by deifyed**
## Description

- Adds `-o/--out` flag to `okctl scaffold application` to enable specifying where to store the application template
- Refactor `okctl create application` to `okctl scaffold application` to align with `okctl scaffold cluster` command
- Minor improvements to `scaffold application` docs

## Motivation and Context

#192 

## How Has This Been Tested?

Manually

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.
- [x] I have updated the release notes (for the next release).

### CODE REVIEW

File: cmd/okctl/scaffoldapplication.go [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+				return err
+			}
+
+			err = scaffold.SaveTemplate(interpolatedResult)
+			if err != nil {
+				return err
+			}
+
+			fmt.Fprintln(o.Out, "Scaffolding successful.")
+			fmt.Fprintf(o.Out, "Edit ./application.yaml to your liking and run okctl apply application %s -f application.yaml\n", opts.Environment)
```
Comment (yngvark) [inline cmd/okctl/scaffoldapplication.go]: Just a check: Is application still in the current directory?

**Review by yngvark — APPROVED** [review]

File: cmd/okctl/scaffoldapplication.go (hunk 1 above)
Comment (deifyed) [reply cmd/okctl/scaffoldapplication.go]: Hmm. This is outdated. Directory is by default the current directory, but can now be specified with --out

Comment (yngvark) [reply cmd/okctl/scaffoldapplication.go]: Yes, and if somebody specify -out, wouldn't this output to the user be invalid? It should be "Edit %s/application.yaml [...]" where %s is the directory specified in --out ?

Comment (deifyed) [reply cmd/okctl/scaffoldapplication.go]: @yngvark the code you pointed to here is outdated.

Relevant code is [here](https://github.com/oslokommune/okctl/pull/264/commits/e1cbe55ec7a0040ff0ed4859543b4ec8ea7c7cae#diff-54bdf823ee965e370ebcfa021a42dba2a92377344927dd2378acc77dfa6bb46cR84) where I've changed the output

**Pull request merged**